build/.docker/docker-compose.yaml

```diff
@@ -22,6 +22,10 @@ services:
       ME_CONFIG_MONGODB_SERVER: mongodb
   application:
     container_name: onboarding-go
-    build: ../
+    ports:
+      - 8080:8080
+    build:
+      context: ../../
+      dockerfile: build/.docker/Dockerfile
     depends_on:
```

Comment:
When running make I got the following error.
```
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "dlv": executable file not found in $PATH: unknown                                                                                                   
```
This is because make does not build the latest version of the container. When I manually ran docker-compose build make worked correctly. Make should be updated to build the container image.

Reply:
I cleaned all images, containers and volumes, but I can't reproduce the error. Looking at your error message, maybe the container could not install delve. I asked Fabricio and Vinicius to run the command make and they don't have this error. It seems is a problem with your local environment.

Reply:
I can't reproduce this either.